sound/soc/intel/boards/bdw-rt5677.c

```diff
@@ -71,6 +71,7 @@ static const struct snd_soc_dapm_route bdw_rt5677_map[] = {
 	{"DMIC L2", NULL, "Local DMICs"},
 	{"DMIC R2", NULL, "Local DMICs"},
 
+#if !IS_ENABLED(CONFIG_SND_SOC_SOF_BROADWELL)
 	/* CODEC BE connections */
 	{"SSP0 CODEC IN", NULL, "AIF1 Capture"},
```

Comment:
This seems incorrect, don't you need these for the DMICs? The last are the only ones related the DSP.

Reply:
Nevermind, I failed to realize the widget was coming from the topology. This will break the WoV though when used in SOF.

Reply:
I have ssp0 defined in the dailink for those

Reply:
> Nevermind, I failed to realize the widget was coming from the topology. This will break the WoV though when used in SOF.

I don't understand what's needed for WoV? you get the data over SPI and then capture over SSP0. Do you have a rough diagram of what the topology should look like?